app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/PocketOnboardTest.java

```diff
@@ -42,8 +43,7 @@
     public static final LocaleTestRule localeTestRule = new LocaleTestRule();
 
     @Rule
-    public ActivityTestRule<MainActivity> mActivityTestRule = new ActivityTestRule<MainActivity>(MainActivity.class,
-            false, false);
+    public ActivityTestRule<MainActivity> mActivityTestRule = new MainActivityTestRule(false, false, false);
```

Comment:
Might be worth removing the Hungarian notation (to be more consistent with the codebase) while touching this file.

Reply:
A good observation but I don't think it's worth the time: I was going to wait to touch up these files until we convert them to kotlin (and use the robot pattern).